src/components/dialogs/move-dialog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { elementType } from '@gridsuite/commons-ui';
+import DirectorySelector from './directory-selector';
+import { useIntl } from 'react-intl';
+
+const MoveDialog = ({ open, onClose, items }) => {
+    const intl = useIntl();
+
+    //TODO : change nbElement
+    return (
+        <>
```

Comment:
why <> ?

Reply:
needs to be removed, was here because of some code that has been removed

Reply:
fixed